### ISSUE TITLE
Error creating user inside docker image

### ISSUE THREAD

**Issue by JavierCVilla** [Kitware/CDash#906]
Using the [cdash-docker image](https://github.com/Kitware/cdash-docker), I get the following series of errors when trying to create a custom user:

```
[Thu Oct 10 07:05:37.561817 2019] [mpm_prefork:notice] [pid 12] AH00163: Apache/2.4.25 (Debian) PHP/7.0.33 configured -- resuming normal operations
[Thu Oct 10 07:05:37.561942 2019] [core:notice] [pid 12] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
127.0.0.1 - - [10/Oct/2019:07:05:47 +0000] "POST /install.php HTTP/1.1" 200 358 "-" "curl/7.52.1"
DEBUG:: PARSED USER ENTRY
DEBUG::   email:        |admin@admin.com|
DEBUG::   disposition:  |ADMIN|
DEBUG::   password:     |admin|
DEBUG::   new password: ||
DEBUG:: PARSED USER INFO
DEBUG::   First Name:  |Admin|
DEBUG::   Last Name:   |User|
DEBUG::   Institution: |Example Foundation|
DEBUG:: BEGIN DUMP OF USER TABLE
DEBUG:: 0:
DEBUG::   admin@admin.com
DEBUG::   disposition: ADMIN
DEBUG::   password: admin
DEBUG::   new pass: 
DEBUG::   First Name: Admin
DEBUG::   Last Name: User
DEBUG::   Institution: Example Foundation
127.0.0.1 - - [10/Oct/2019:07:05:47 +0000] "POST /user.php HTTP/1.1" 200 18833 "-" "curl/7.52.1"
DEBUG:: ||
[Thu Oct 10 07:05:48.148103 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Warning:  file_get_contents(login.xsl): failed to open stream: No such file or directory in /var/www/cdash/include/common.php on line 60
[Thu Oct 10 07:05:48.148152 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Stack trace:
[Thu Oct 10 07:05:48.148174 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.148185 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.148204 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.148220 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.148240 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.148261 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   6. file_get_contents() /var/www/cdash/include/common.php:60
[Thu Oct 10 07:05:48.148290 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Warning:  DOMDocument::loadXML(): Empty string supplied as input in /var/www/cdash/include/common.php on line 60
[Thu Oct 10 07:05:48.148305 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Stack trace:
[Thu Oct 10 07:05:48.148318 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.148334 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.148346 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.148366 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.148386 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.148421 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   6. DOMDocument->loadXML() /var/www/cdash/include/common.php:60
[Thu Oct 10 07:05:48.148527 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Warning:  XSLTProcessor::importStylesheet(): compilation error in /var/www/cdash/include/common.php on line 63
[Thu Oct 10 07:05:48.148541 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Stack trace:
[Thu Oct 10 07:05:48.148551 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.148563 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.148574 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.148584 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.148594 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.148608 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   6. XSLTProcessor->importStylesheet() /var/www/cdash/include/common.php:63
[Thu Oct 10 07:05:48.148625 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Warning:  XSLTProcessor::importStylesheet(): xsltParseStylesheetProcess : empty stylesheet in /var/www/cdash/include/common.php on line 63
[Thu Oct 10 07:05:48.148635 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Stack trace:
[Thu Oct 10 07:05:48.148645 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.148656 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.148668 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.148678 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.148690 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.148703 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   6. XSLTProcessor->importStylesheet() /var/www/cdash/include/common.php:63
[Thu Oct 10 07:05:48.148737 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Warning:  XSLTProcessor::transformToXml(): No stylesheet associated to this object in /var/www/cdash/include/common.php on line 73
[Thu Oct 10 07:05:48.148746 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP Stack trace:
[Thu Oct 10 07:05:48.148759 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.148771 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.148784 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.148799 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.148811 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.148831 2019] [:error] [pid 15] [client 127.0.0.1:34300] PHP   6. XSLTProcessor->transformToXml() /var/www/cdash/include/common.php:73
127.0.0.1 - - [10/Oct/2019:07:05:48 +0000] "GET /ajax/findusers.php?search=admin@admin.com HTTP/1.1" 200 562 "-" "curl/7.52.1"
DEBUG:: CREATING USER: admin@admin.com
127.0.0.1 - - [10/Oct/2019:07:05:48 +0000] "POST /manageUsers.php HTTP/1.1" 200 5207 "-" "curl/7.52.1"
[Thu Oct 10 07:05:48.690266 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Warning:  file_get_contents(login.xsl): failed to open stream: No such file or directory in /var/www/cdash/include/common.php on line 60
[Thu Oct 10 07:05:48.690326 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Stack trace:
[Thu Oct 10 07:05:48.690348 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.690368 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.690378 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.690391 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.690402 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.690412 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   6. file_get_contents() /var/www/cdash/include/common.php:60
[Thu Oct 10 07:05:48.690435 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Warning:  DOMDocument::loadXML(): Empty string supplied as input in /var/www/cdash/include/common.php on line 60
[Thu Oct 10 07:05:48.690445 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Stack trace:
[Thu Oct 10 07:05:48.690461 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.690472 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.690481 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.690504 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.690517 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.690532 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   6. DOMDocument->loadXML() /var/www/cdash/include/common.php:60
[Thu Oct 10 07:05:48.690650 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Warning:  XSLTProcessor::importStylesheet(): compilation error in /var/www/cdash/include/common.php on line 63
[Thu Oct 10 07:05:48.690665 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Stack trace:
[Thu Oct 10 07:05:48.690676 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.690689 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.690700 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.690710 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.690723 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.690741 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   6. XSLTProcessor->importStylesheet() /var/www/cdash/include/common.php:63
[Thu Oct 10 07:05:48.690756 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Warning:  XSLTProcessor::importStylesheet(): xsltParseStylesheetProcess : empty stylesheet in /var/www/cdash/include/common.php on line 63
[Thu Oct 10 07:05:48.690766 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Stack trace:
[Thu Oct 10 07:05:48.690776 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.690788 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.690799 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.690812 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.690836 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.690851 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   6. XSLTProcessor->importStylesheet() /var/www/cdash/include/common.php:63
[Thu Oct 10 07:05:48.690889 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Warning:  XSLTProcessor::transformToXml(): No stylesheet associated to this object in /var/www/cdash/include/common.php on line 73
[Thu Oct 10 07:05:48.690902 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP Stack trace:
[Thu Oct 10 07:05:48.690915 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   1. {main}() /var/www/cdash/public/ajax/findusers.php:0
[Thu Oct 10 07:05:48.690925 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   2. include() /var/www/cdash/public/ajax/findusers.php:19
[Thu Oct 10 07:05:48.690935 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   3. LoginForm() /var/www/cdash/public/login.php:66
[Thu Oct 10 07:05:48.690944 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   4. generate_XSLT() /var/www/cdash/include/login_functions.php:445
[Thu Oct 10 07:05:48.690959 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   5. xslt_process() /var/www/cdash/include/common.php:137
[Thu Oct 10 07:05:48.690975 2019] [:error] [pid 17] [client 127.0.0.1:34308] PHP   6. XSLTProcessor->transformToXml() /var/www/cdash/include/common.php:73
127.0.0.1 - - [10/Oct/2019:07:05:48 +0000] "GET /ajax/findusers.php?search=admin@admin.com HTTP/1.1" 200 336 "-" "curl/7.52.1"
Warning: unable to create user account admin@admin.com: unknown error
``` 

As a result, I cannot log in to the administration area. I also tried using the default `rootadmin@docker.container` mail and `CDASH_ROOT_ADMIN_PASS` but it does not let me in.  

After building the docker image, I'm using the following command to start the container:

```
docker run  -e CDASH_CONFIG="$(cat local-configuration.php)" -e CDASH_STATIC_USERS="$(cat static-users.php)" -e CDASH_ROOT_ADMIN_PASS=admin -e DEBUG=1 -it -p 8080:80 kitware/cdash-docker 
```

where `local-configuration.php` contains my custom configuration and `static-users.php` has the information to create the `admin` user.

**Comment by bryonbean** [comment]
Likely same issue as #887.

**Comment by JavierCVilla** [comment]
@bryonbean any workaround?

**Comment by bryonbean** [comment]
@JavierCVilla I will look into this sometime this week and keep you posted (possibly next Monday).

**Comment by galabovaa** [comment]
Is there any progress on that issue? I run into the same one.

**Comment by zackgalbreath** [comment]
This was fixed in #986. Please feel free to reopen this issue if it still doesn't work for you.
